lib/jekyll/liquid_renderer/table.rb

```diff
@@ -75,18 +80,23 @@ def data_for_table(num_of_rows)
         sorted = @stats.sort_by { |_, file_stats| -file_stats[:time] }
         sorted = sorted.slice(0, num_of_rows)
 
-        table = [%w(Filename Count Bytes Time)]
+        gauges = [:count, :bytes, :time]
```

Comment:
`gauges` should be a constant here, IMO.

Reply:
This entire method is going to be called just once per invocation.. so is this really necessary?

Reply:
Yes. It is.

It is not the number of times that it is called that makes it a constant, but the type of thing it is. We are not changing the array in this code, therefore it is a constant.